protos/storage.proto

```diff
@@ -105,7 +105,8 @@ message QueryResponse {
     JSON = 1;
   }
   SchemaType schema_type = 2;
-  string schema_content = 3;
+  string schema_content = 3; // versions after table_schema will be deprecated
```

Comment:
Remove `schema_type` and `schema_content`,  no need to consider compatibility.

Reply:
get.
rm `schema_content` already, and rename `schema_type` to `serialization_type` for avoid ambiguity